src/Cake.Common/Tools/NuGet/Pack/NuGetPackSettings.cs

```diff
@@ -65,6 +65,12 @@ public sealed class NuGetPackSettings : ToolSettings
         /// <value>The Nuspec version.</value>
         public string Version { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Nuspec version suffix.
+        /// </summary>
+        /// <value>The Nuspec version suffix.</value>
+        public string VersionSuffix { get; set; }
```

Comment:
I think this should be named `Suffix` to be consistent with the underlying tool.